Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Groundrunner.xml

```diff
@@ -20,13 +20,14 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_Groundrunner"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>3</ArmorRating_Blunt>
+					<ArmorRating_Blunt>6</ArmorRating_Blunt>
 				</value>
 			</li>
 			
 			<li Class="PatchOperationAdd">
 				<xpath>/Defs/ThingDef[defName="AA_Groundrunner"]/statBases</xpath>
 				<value>
+					<ArmorRating_Sharp>1</ArmorRating_Sharp>

```

Comment:
It's a hybrid bear/mole, why does it have any armor at all?! The small amount of blunt armor it does have represents a general sturdiness and thick layer of absorbent flesh, essentially.